odd-platform-ui/src/components/DataEntityDetails/Overview/OverviewMetadata/MetadataItem/MetadataItem.tsx

```diff
@@ -66,49 +60,61 @@ const MetadataItem: React.FC<MetadataItemProps> = ({
     });
 
   let metadataVal;
+
   try {
     switch (metadataItem.field.type) {
       case MetadataFieldType.BOOLEAN:
-        metadataVal = <BooleanFormatted value={metadataItem.value} />;
+        metadataVal = metadataItem.value === 'true' ? 'Yes' : 'No';
```

Comment:
We have similar boolean value representation across all the app, which could be changed easily changed by updating `BooleanFormatted` component, so I don't think removing it is a good idea.
I believe that adding a `valToCopy` and keep `valToShow`(metadataVal) in a separate variable is a more flexible solution.